utils.go

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package virtcontainers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+func fileCopy(srcPath, dstPath string) error {
+	if srcPath == "" {
+		return fmt.Errorf("Source path cannot be empty")
+	}
+
+	if dstPath == "" {
+		return fmt.Errorf("Destination path cannot be empty")
+	}
+
+	srcInfo, err := os.Stat(srcPath)
+	if err != nil {
+		return err
+	}
+
+	srcContent, err := ioutil.ReadFile(srcPath)
+	if err != nil {
+		return err
+	}
+
+	if err := os.RemoveAll(dstPath); err != nil {
```

Comment:
I'd be tempted to call `RemoveAll()` before `ReadFile()` since if you cannot remove the destination, there isn't much point in reading the source (first).